R/RLearner_classif_randomForest.R

```diff
@@ -52,6 +52,14 @@ predictLearner.classif.randomForest = function(.learner, .model, .newdata, ...)
 }
 
 #' @export
+getOOBPredsLearner.classif.randomForest = function(.learner, .model) {
+  if (.learner$predict.type == "response") {
+    unname(.model$learner.model$predicted)
+  } else {
+   .model$learner.model$votes
+  }
+}
+
```

Comment:
Really? Ups, I thought it was missing before `getFeatureImportanceLearner.classif.randomForest`